src/app/components/pages/Admin.tsx

```diff
@@ -91,18 +92,17 @@ export const Admin = ({ user }: { user: RegisteredUserDTO }) => {
                   until={displayVersion !== null}
                   thenRender={() => {
                     return (
-                      displayVersion !== null &&
-                      updateState != ContentVersionUpdatingStatus.UPDATING && (
-                        <RS.Form onSubmit={startVersionUpdate}>
-                          <RS.InputGroup>
-                            <RS.Input
+                      displayVersion !== null && (
+                        <Form>
+                          <InputGroup>
+                            <Input
```

Comment:
I assume we're retaining this as a way to view the current SHA? Does it still need to be an input if we're not actually allowing it to be set?

Reply:
Yes because this is for the "Preview Changes" - you input a new SHA and then preview changes and it shows the difference between live version and new one